docs/writing-policies/metadata.md

```diff
@@ -0,0 +1,105 @@
+---
+sidebar_label: "Policy metadata"
+title: "Policy metadata"
+---
+
+# Policy metadata
+
+The Kubewarden metadata file, `metadata.yaml`, serves as a vital configuration
+file that contains important information and settings related to the policies
+used within Kubewarden. This documentation aims to provide an overview of the
+purpose and usage of the metadata file.
+
+The policy `metadata.yaml` file contains defaults for the policy, in addition
+to metadata such as author and description, set by the policy author.  The file
+is used by the `kwctl annonate` command to, as the name suggests, annotates the
+`.wasm` file containing the policy. Therefore, all the relevant information required to run
+the policy will be available. See more how to annotate the policy in the
+[Distributing Policies](../distributing-policies.md) guide
+
+When policy users want to use a policy, they generate a YAML manifest using
+`kwctl scaffold`. This command reads the policy metadata embedded in the
+shipped Wasm module, performs sanity checks, and returns a YAML manifest that
+the author can use as-is or adapt to their needs.
+
+As a policy author, you can adapt the `metadata.yaml` file provided during the
+scaffolding of your policy.
+
+See the following example of a `metadata.yaml`:
+
+
+```yaml
+rules:
+- apiGroups: [""]
+  apiVersions: ["v1"]
+  resources: ["pods"]
+  operations: ["CREATE"]
+mutating: false
+contextAwareResources: []
+executionMode: kubewarden-wapc
+backgroundAudit: true
+annotations:
+  # artifacthub specific:
+  io.artifacthub.displayName: Policy Name
+  io.artifacthub.resources: Pod
+  io.artifacthub.keywords: pod, cool policy, kubewarden
+  io.kubewarden.policy.ociUrl: ghcr.io/myorg/my-policy
+  # kubewarden specific:
+  io.kubewarden.policy.title: My policy
+  io.kubewarden.policy.description: Short description
+  io.kubewarden.policy.author: myself
+  io.kubewarden.policy.url: https://github.com/yourorg/my-policy
+  io.kubewarden.policy.source: https://github.com/yourorg/my-policy
+  io.kubewarden.policy.license: Apache-2.0
+  # The next two annotations are used in the policy report generated by the
+  # Audit scanner. Severity indicates policy check result criticality and
+  # Category indicates policy category. See more here at docs.kubewarden.io
+  io.kubewarden.policy.severity: medium
+  io.kubewarden.policy.category: Resource validation
+```
+
+
+**1. Enabling Background Audit Checks:**
+
+The metadata file includes a flag, `backgroundAudit`,  that enables the
+background audit checks for a specific policy. By default, this flag is set to
+`true`. There are some policies where, due some audit scanner limitations, this
+field should be set to `false` tough.  See more about the audit scanner
+limitations that policy author should have in mind when enabling the
+`backgroundAudit` in its dedicated [docs page](../explanations/audit-scanner)
```

Comment:
```suggestion
`true`.

There are some policies that, due to the way they work or to the type
of events they are interested about, should have this field set to `false`.
More information can be found inside of the
[audit scanner documentation](../explanations/audit-scanner),
respectively under the limitations section.
```

It would be nice to have the link point straight to the limitation section. I think this is doable... something like `../explanations/audit-scanner#limitations`?

Reply:
I do not know. I'll check in the docusaurus docs.